chore: remove Celer bridge integration

saledjenic opened the pull request:
- Deleted Celer bridge ABI and Go binding files.
- Removed related configurations and feature flags from wallet and protocol settings.
- Cleaned up wallet service code by eliminating references to Celer bridge processing.
- Updated path processor to remove Celer bridge transaction handling.

saledjenic committed:
chore: remove multi-transaction handling from wallet services

- Simplified transaction management by removing multi-transaction ID from `SendTransactionWithSignature` and related methods.
- Deprecated endpoints removed `WatchTransactionByChainID`, `ProceedWithTransactionsSignatures` and `GetMultiTransactions`.
- Updated transaction handling in various services to streamline the process.
- Cleaned up related tests to reflect these changes.

saledjenic committed:
chore: remove Celer bridge integration

- Deleted Celer bridge ABI and Go binding files.
- Removed related configurations and feature flags from wallet and protocol settings.
- Cleaned up wallet service code by eliminating references to Celer bridge processing.
- Updated path processor to remove Celer bridge transaction handling.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (18)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 4de1cfb9 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6938/1/) | 2025-09-18 06:33:32 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-063007-4de1cf-pr6938.zip) |
| :heavy_check_mark: | 4de1cfb9 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6938/1/) | 2025-09-18 06:34:09 | ~4 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-063002-4de1cf-pr6938.zip) |
| :heavy_check_mark: | 4de1cfb9 | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6938/1/) | 2025-09-18 06:35:31 | ~5 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-063013-4de1cf-pr6938.zip) |
| :heavy_check_mark: | 4de1cfb9 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6938/1/) | 2025-09-18 06:40:17 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-063004-4de1cf-pr6938.zip) |
| :heavy_check_mark: | 4de1cfb9 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6938/1/) | 2025-09-18 06:40:22 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6938/1//consoleText) |
| :heavy_check_mark: | 4de1cfb9 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6938/1/) | 2025-09-18 06:55:52 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6938/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 1f2e16f7 | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6938/2/) | 2025-09-18 08:40:37 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-083709-1f2e16-pr6938.zip) |
| :heavy_check_mark: | 1f2e16f7 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6938/2/) | 2025-09-18 08:40:48 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-083702-1f2e16-pr6938.zip) |
| :heavy_check_mark: | 1f2e16f7 | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6938/2/) | 2025-09-18 08:42:46 | ~5 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-083715-1f2e16-pr6938.zip) |
| :heavy_check_mark: | 1f2e16f7 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6938/2/) | 2025-09-18 08:48:03 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6938/2//consoleText) |
| :heavy_check_mark: | 1f2e16f7 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6938/2/) | 2025-09-18 08:48:35 | ~11 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-083704-1f2e16-pr6938.zip) |
| :heavy_check_mark: | 1f2e16f7 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6938/2/) | 2025-09-18 09:03:33 | ~26 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6938/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | b992e202 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6938/3/) | 2025-09-18 13:17:34 | ~5 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-131226-b992e2-pr6938.zip) |
| :heavy_check_mark: | b992e202 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6938/3/) | 2025-09-18 13:17:43 | ~5 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-131231-b992e2-pr6938.zip) |
| :heavy_check_mark: | b992e202 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6938/3/) | 2025-09-18 13:35:21 | ~22 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-131221-b992e2-pr6938.zip) |
| :heavy_check_mark: | b992e202 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6938/3/) | 2025-09-18 13:42:55 | ~30 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-131221-b992e2-pr6938.zip) |
| :heavy_check_mark: | b992e202 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6938/3/) | 2025-09-18 13:45:38 | ~33 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6938/3//consoleText) |
| :heavy_check_mark: | b992e202 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6938/3/) | 2025-09-18 14:02:51 | ~50 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6938/3//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 8848c01e | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6938/4/) | 2025-09-22 14:25:46 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-142237-8848c0-pr6938.zip) |
| :heavy_check_mark: | 8848c01e | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6938/4/) | 2025-09-22 14:26:04 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-142241-8848c0-pr6938.zip) |
| :heavy_check_mark: | 8848c01e | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6938/4/) | 2025-09-22 14:28:11 | ~5 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-142246-8848c0-pr6938.zip) |
| :heavy_check_mark: | 8848c01e | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6938/4/) | 2025-09-22 14:32:40 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6938/4//consoleText) |
| :heavy_check_mark: | 8848c01e | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6938/4/) | 2025-09-22 14:32:50 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250922-142238-8848c0-pr6938.zip) |
| :heavy_check_mark: | 8848c01e | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6938/4/) | 2025-09-22 14:48:50 | ~26 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6938/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | c5ce72a3 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6938/5/) | 2025-09-22 14:29:57 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-142556-c5ce72-pr6938.zip) |
| :heavy_check_mark: | c5ce72a3 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6938/5/) | 2025-09-22 14:31:15 | ~5 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-142609-c5ce72-pr6938.zip) |
| :heavy_check_mark: | c5ce72a3 | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6938/5/) | 2025-09-22 14:33:34 | ~5 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-142814-c5ce72-pr6938.zip) |
| :heavy_check_mark: | c5ce72a3 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6938/5/) | 2025-09-22 14:42:46 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6938/5//consoleText) |
| :heavy_check_mark: | c5ce72a3 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6938/5/) | 2025-09-22 14:43:54 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250922-143254-c5ce72-pr6938.zip) |
| :heavy_check_mark: | c5ce72a3 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6938/5/) | 2025-09-22 15:15:21 | ~26 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6938/5//consoleText) |

saledjenic committed:
chore: simplify transaction handling by removing multi-transaction ID

- Updated multiple services and tests to eliminate multi-transaction ID from `SendTransactionWithSignature` and related methods.
- Adjusted transaction management logic across the code base.
- Cleaned up related tests.

saledjenic committed:
chore: remove multi-transaction ID references and clean up code

- Eliminated multi-transaction ID from various structures and functions across the wallet service.
- Updated related queries and database interactions to reflect the removal of multi-transaction handling.
- Cleaned up related tests.

saledjenic committed:
chore: remove unused methods from path processors

- Eliminated `Send` and `BuildTransaction` methods.
- Updated related tests to reflect the removal of these methods.
- Codebase cleaned up accordingly.

dlipicar commented on services/wallet/router/pathprocessor/multipath_processor.go:
```diff
@@ -24,8 +23,8 @@ func (t *MultipathProcessorTxArgs) Value() *big.Int {
 		return t.HopTx.Amount.ToInt()
-	} else if t.CbridgeTx != nil {
-		return t.CbridgeTx.Amount.ToInt()
+	} else if t.SwapTx != nil {
```
this looks weird, was Swap missing from this handler?

saledjenic replied:
Not at all, it should not be here.

saledjenic replied:
Most likely somehow remainded when I was splitting commits for PR. Will remove those lines, even `MultipathProcessorTxArgs` is completely removed in the follow-up PR.
